basilisk/src/ast/stack.c

```
@@ -0,0 +1,66 @@
+  void * (* push) (Stack *, void *);
+  void * data;
+};
+
+Stack * stack_new (int size)
+{
+  Stack * s = calloc (1, sizeof (Stack));
+  s->size = size;
+  return s;
+}
+
+void stack_push (Stack * s, void * p)
```

Clang (14.0.6)

[warning] parameter name 's' is too short, expected at least 3 characters

(readability-identifier-length)